explainers/wpub-explainer.md

```diff
@@ -136,33 +142,21 @@ The key questions are [1] identifying the "bounds" of the publication, [2] defin
 
 ```
 
-The web publications spec has chosen to use an explicit list of resources. Anything that is not part of the `resources` or `readingOrder` manifest members is considered to be outside the web publication. The group considered things like `scope` and URL patterns, but felt that an explicit list was simpler. 
-
 ### 2. Sequence of primary resources
 
-- HTML can express ordering of resources via `rel=prev` and `rel=next`, but these do not have UI support from the browsers (with notably rare exceptions).
+Web publications use an explicit list in the manifest to define the reading sequence &#8212; via the `readingOrder` member &#8212; which is again similar to EPUB and its use of a `spine1` element. Other methods considered to achieve sequencing have included:
 
-- HTML can also express an ordered list of links with the `nav` element. 
+- `rel=prev` and `rel=next` in HTML, but these do not have UI support from the browsers (with notably rare exceptions) and are not available for most non-HTML media types.
 
-- Transclusion can also define an ordering of the transcluded components. 
+- identifying sequence via a list of links in a `nav` element (e.g., the table of contents), but not all resources may be listed in the table of contents
 
-- EPUB uses the `spine` element in the XML package file to determine a primary reading order.
-
-Once again, web publications use an explicit list outside of the content itself, via the `readingOrder` member of the manifest. 
+- the DOM order of transcluded documents
 
 ### 3. Metadata
 
-- There are many ways of embedding metadata in individual HTML files
-
-- EPUB includes metadata in the XML package file
-
-
-- Metadata about a web application can be expressed in the web application manifest file. 
-
-- With transclusion, one could define metadata in the parent file to apply to the whole.
-
-Web publication metadata is expressed in the manifest using the schema.org vocabulary, with some extensions. The metadata is serialized as JSON-LD, one of the three syntaxes accepted by schema.org (the others being microdata and RDFa). JSON-LD has also the advantage of being easy to use for both client and server side processing.
+Web publication metadata is expressed in the manifest using the schema.org vocabulary, with some extensions. The metadata is serialized as JSON-LD, one of the three syntaxes accepted by schema.org (the others being microdata and RDFa). JSON-LD has also the advantage of being easy to use for both client and server side processing. As previously, this method is similar to the EPUB approach.
 
+Alternatively, it is possible to embed metadata directly in the header and/or content of HTML files, which could include the parent of a set of transcluded files so that it applies to all the sub-documents.
```

Comment:
Maybe worth emphasizing at this point that, at this time, most of the schema.org processors (if not all) rely on the embedded JSON-LD approach rather than the separate file. The reason this is worth emphasizing is that the embedding approach was rejected for the Web Application Manifest (after some discussions), i.e., our decision is in contrast to what they decided to do.